CHANGELOG

```diff
@@ -21,6 +21,7 @@ Enhancements
   * add timer for block-IO and block-compute
   * store block information in `_block` attribute (Issue #89)
   * add parallel density class (Issue #8)
+  * add `get_cdf` function to `pmda.rdf.InterRDF`
```

Comment:
add `get_cdf` function to `pmda.rdf.InterRDF` to calculate cumulative distribution function